xapian-core/include/xapian/weight.h

```diff
@@ -467,6 +467,33 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
        tfidf weight. */
     std::string normalizations;
```

Comment:
Is this still used anywhere?

Reply:
Yes ,In switch I added.

Reply:
I don't think you need to though — since that switch is in a constructor, you could just use the formal parameter (`normals`) directly.

Reply:
Updated to use normals